pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -69,6 +69,8 @@ type ReplicationChannel struct {
 type ReplicationChannelConfig struct {
 	SourceRetryCount   uint `json:"sourceRetryCount,omitempty"`
 	SourceConnectRetry uint `json:"sourceConnectRetry,omitempty"`
+	Ssl                bool   `json:"ssl,omitempty"`
+	Ca                 string `json:"ca,omitempty"`	
```

Comment:
please use all caps for these variables. see https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
done